bindings/electron/src/meths.rs

```diff
@@ -6521,6 +6521,66 @@ fn client_stop(mut cx: FunctionContext) -> JsResult<JsPromise> {
     Ok(promise)
 }
 
+// get_default_config_dir
+fn get_default_config_dir(mut cx: FunctionContext) -> JsResult<JsPromise> {
+    let ret = libparsec::get_default_config_dir();
+    let js_ret = JsString::try_new(&mut cx, {
+        let custom_to_rs_string = |path: std::path::PathBuf| -> Result<_, _> {
+            path.into_os_string()
+                .into_string()
+                .map_err(|_| "Path contains non-utf8 characters")
+        };
+        match custom_to_rs_string(ret) {
+            Ok(ok) => ok,
+            Err(err) => return cx.throw_type_error(err),
+        }
+    })
+    .or_throw(&mut cx)?;
+    let (deferred, promise) = cx.promise();
+    deferred.resolve(&mut cx, js_ret);
+    Ok(promise)
```

Comment:
This code seems identical in the 3 functions, maybe there is a way to extract it into a function :
```rust
fn from_dir(path: std::path::PathBuf, mut cx: FunctionContext) -> JsResult<JsPromise>
```

Reply:
this code is autogenerated (and differs between electron, web and android in there future), so I'd rather stay KISS there :smiley: